src/guild/ShoukakuPlayer.js

```diff
@@ -363,16 +363,21 @@ class ShoukakuPlayer extends EventEmitter {
     /**
      * Tries to resume your player, a use case for this is when you do ShoukakuPlayer.connection.attemptReconnect()
      * @memberOf ShoukakuPlayer
+     * @param {Object} [options={}] Override any options that are passed to playTrack when resuming playback.
```

Comment:
Document the options, which you can copy paste from playTrack options documentation

Reply:
Could you elaborate?

Do you mean these?
```js
{ noReplace?: boolean, pause?: boolean, startTime?: number, endTime?: number }
```

Reply:
Or do I add the same comment as there is on playTrack?

```
     * @param {Object} [options={}] Used if you want to put a custom track start or end time
```

Reply:
https://github.com/Deivu/Shoukaku/blob/17283ad9e0a17185070f289886510423cb19e6ab/src/guild/ShoukakuPlayer.js#L129 
This documentation on options you could just put it at that on documentation

Reply:
Alright will do that